pkg/simulation/simulation.go

```diff
@@ -30,12 +30,21 @@ func (s *Simulation) FetchData(url string) {
 	s.ResetTrams()
 }
 
-func (s *Simulation) GetTramIDs() (result []int) {
-	result = make([]int, len(s.trams))
-	for i, tram := range s.trams {
-		result[i] = tram.id
-	}
+type TramIdentifier struct {
+	ID    int    `json:"id"`
+	Route string `json:"route"`
+}
 
+func (s *Simulation) GetTramIDs() (result []TramIdentifier) {
+	result = make([]TramIdentifier, len(s.trams))
```

Comment:
This way we ensure that the slice has 0 elements initially and the capacity to hold all expected elements without the need for reallocation:
```suggestion
	result = make([]TramIdentifier, 0, len(s.trams))
```